.github/workflows/_link_check.yml

```diff
@@ -55,3 +55,29 @@ jobs:
           echo "Or add \`@lint-ignore\` somewhere on the same line as the reference you want to skip checking."
           exit 1
         }
+
+  lint-file-size:
+    if: ${{ github.event_name == 'pull_request' }}
+    uses: pytorch/test-infra/.github/workflows/linux_job_v2.yml@main
+    with:
+      runner: linux.2xlarge
```

Comment:
do we have smaller runners we can use? this isn't particularly resource-intensive

Reply:
Tried linux.large and docker image pulling failed. Bump it to 2x